tapfreighter/wallet.go

```diff
@@ -385,23 +394,34 @@ func (f *AssetWallet) FundPacket(ctx context.Context,
 		AssetID:  &fundDesc.ID,
 		MinAmt:   1,
 	}
-	eligibleCommitments, err := f.cfg.CoinSelector.ListEligibleCoins(
-		ctx, constraints,
-	)
-	if err != nil {
-		return nil, fmt.Errorf("unable to complete coin selection: %w",
-			err)
-	}
 
-	log.Infof("Identified %v eligible asset inputs for send of %d to %x",
-		len(eligibleCommitments), fundDesc.Amount, fundDesc.ID[:])
+	var selectedCommitments []*AnchoredCommitment
```

Comment:
Perhaps we also want to have a wrapper type here that makes it easier to release all the coin in a single go?